src/pages/docs/packaging-applications/build-servers/github-actions.mdx

```diff
@@ -196,6 +198,92 @@ jobs:
 | `packages`            | Used by the Push Packages action. A list of packages to push to Octopus Deploy. Add each item on a new line.     |
 | `packages`            | Used by the Create Release action. A list of packages to be used in the Release. Add each item on a new line.    |
 
+## Handling build information
+
+<BuildInformationInReleaseNotesWithArc />
+
+### Example workflow - Working with packages and build information
+
+```yaml
+# .github/workflows/hello-octopus-packages-and-build-information.yml
+name: Hello Octopus Packages and Build Information
+
+on:
+  workflow_dispatch:
+
+jobs:
+  octopus-packages:
+    runs-on: ubuntu-latest
+
+    permissions:
+      id-token: write # Required by login action
+
+    env:
+      OCTOPUS_SPACE: 'Outer Space'
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      # Action to Login to Octopus Deploy
+      - name: Login to Octopus Deploy 🐙
+        uses: OctopusDeploy/login@v1
+        with: 
+          server: ${{ secrets.SERVER }}
+          service_account_id: a1a1a1a1-b2b2-c3c3-d4d4-e5e5e5e5e5e5
+      
+      # Action to Create a Zip Package
+      - name: Create a Zip package 🐙
+        uses: OctopusDeploy/create-zip-package-action@v3
+        with:
+          package_id: 'HelloPackage'
+          version: '1.0.0'
+          output_folder: './packages/'
+          base_path: './src/files-to-package/'
+          files: |
+            **/*.*
+      
+      # Action to Push Packages to Octopus Deploy
+      - name: Push a package to Octopus Deploy 🐙
+        uses: OctopusDeploy/push-package-action@v3
+        with:
+          packages: |
+            packages/**/*.zip
+      
+      # Action to Push Build Information to Octopus Deploy
+      - name: Push build information to Octopus Deploy 🐙
+        uses: OctopusDeploy/push-build-information-action@v3
+        with:
+          packages: |
+            HelloPackage
+          version: '1.0.0'
```

Comment:
Isn't this example the opposite of the recommendation?

Reply:
yes, **but** this is the order for "normal" (non-ARC) scenario and is what the other docs pages show.